sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncClient.java

```diff
@@ -406,10 +406,6 @@ boolean isContentResponseOnWriteEnabled() {
      * @return a {@link CosmosPagedFlux} containing one or several feed response pages of read databases or an error.
      */
     public CosmosPagedFlux<CosmosDatabaseProperties> queryDatabases(String query, CosmosQueryRequestOptions options) {
```

Comment:
Should we not have Cosmos changes?

Reply:
you are reverting our code!

FYI @simplynaveen20

Reply:
It's a merge conflict - I'm trying to resolve

Reply:
It seems like these changes were made after I merged, merging again now

Reply:
Synced and cosmos changes are gone